src/initializers/log4js/json-appender.ts

```diff
@@ -23,11 +23,11 @@ export const createErrorLog = logEvent => {
   };
 };
 
-export const createValidLog = logEvent => {
+export const createValidLog = (layout, logEvent) => {
```

Comment:
What if you don't pass layout?

Reply:
This method is exported only for testing purposes. So layout will be passed from json appender initialisation